pkg/agent/manager/cache/cache_test.go

```diff
@@ -5,6 +5,7 @@ import (
 	"crypto/elliptic"
 	"crypto/rand"
 	"crypto/x509"
+	"github.com/sirupsen/logrus"
```

Comment:
nit: this include belongs below and not with the stdlib includes

Reply:
:+1: